### PR TITLE
Add FromForm and IFormCollection to precedence list

### PR DESCRIPTION
**Pull request by tdykstra**
Follow-up to #29417 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|

### CODE REVIEW

File: aspnetcore/fundamentals/minimal-apis/includes/parameter-binding7.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Supported binding sources:
 * Services provided by dependency injection
 * Custom
 
-Binding from form values is ***not*** natively supported in .NET.
+Binding from form values is ***not*** natively supported in .NET 6 and 7.
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/minimal-apis/includes/parameter-binding7.md:13]: Is it worth saying, it's available in 8+ and append ?`view=aspnetcore-8.0&preserve-view=true` to the link?

Probably not worth it as 8 will be out soon and is LTS and the default version.

**Review by Rick-Anderson — APPROVED** [review]

File: aspnetcore/fundamentals/minimal-apis/includes/parameter-binding7.md (hunk 1 above)
Comment (Rick-Anderson) [reply aspnetcore/fundamentals/minimal-apis/includes/parameter-binding7.md:13]: Are you taking care of this in What's new in 8?

Comment (tdykstra) [reply aspnetcore/fundamentals/minimal-apis/includes/parameter-binding7.md:13]: > Probably not worth it as 8 will be out soon and is LTS and the default version.

Agreed - people looking at the old version 7 of the doc will know to check out version 8.

Comment (tdykstra) [reply aspnetcore/fundamentals/minimal-apis/includes/parameter-binding7.md:13]: > Are you taking care of this in What's new in 8?

Yes

**Pull request merged**